.eslintrc.js

```diff
@@ -49,6 +50,10 @@ module.exports = {
     // allow debugger during development
     'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
     'brace-style': ["error", "stroustrup", { "allowSingleLine": true }],
+    // disable linebreak-style rule
+    'linebreak-style': 0,
+    "global-require": 0,
+    "eslint linebreak-style": [0, "error", "windows"]
```

Comment:
Please remove this line. `eslint linebreak-style` is not the name of any eslint rule so this effectively does nothing.